lib/public/views/lhcPeriods/ActiveColumns/lhcPeriodsActiveColumns.js

```diff
@@ -84,7 +84,7 @@ export const lhcPeriodsActiveColumns = {
         name: 'Beam Type',
         visible: true,
         sortable: true,
-        format: (beamType) => beamType || '-',
+        format: (beamType) => beamType.length > 0 ? beamType : '-',
```

Comment:
```suggestion
        format: (beamType) => beamType || '-',
```

Reply:
no

Reply:
no, beamType is not a string but array

Reply:
If it is an array, it should be `beamTypes` ? Or `beamTypeList` ?

Reply:
well, in 99.99% there is one entry, but yes, `beamTypes` will be better

Reply:
applied